distributed/bokeh/scheduler_html.py

```diff
@@ -163,6 +164,23 @@ def get(self):
         self.write(result)
 
 
+class PrometheusHandler(RequestHandler):
+    # Construct Prometheus metrics
+    # https://prometheus.io/docs/introduction/overview/
+    def get(self):
+        workers = Gauge('workers_total',
```

Comment:
Because prometheus functions are only used a couple times in the codebase I'd prefer that we keep the explicit namespace and use the full `prometheus_client.Gauge` name.  That way future maintainers will have more context about what a Gauge object is without having to dereference and find the import.

Reply:
makes sense, fixed scope and made explicit.